deployment/urunc-deploy/urunc-cleanup/overlays/k3s/mount_k3s_conf.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
```

Comment:
This file seems identical for urucn-cleanup and urunc-deploy. Would it be possible to merge them?

Reply:
While they are similar, they have some differences in their structure (spec, selectors, etc). I am not sure what changes we should make to use avoid duplication.

Reply:
Ok, we can do that in another iteration if required.